### PR TITLE
Add podaacpy recipe

### PR DESCRIPTION
**Pull request by agoodm**
Just to clarify, @lewismc already expressed his desire to be added as maintainer [here](https://github.com/lewismc/podaacpy/issues/83).

### CODE REVIEW

**Comment by lewismc** [comment]
ack @agoodm thanks

File: recipes/podaacpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - beautifulsoup4
+    - requests
+
+test:
+  imports:
+    - podaac
+
+about:
+  home: https://github.com/lewismc/podaacpy
+  license: Apache 2.0
```
Comment (jakirkham) [inline recipes/podaacpy/meta.yaml]: Would suggest adding `license_family: Apache`.

**Comment by jakirkham** [comment]
Basically LGTM. One minor comment, but it should be easy to fix.

File: recipes/podaacpy/meta.yaml (hunk 1 above)
Comment (agoodm) [inline recipes/podaacpy/meta.yaml]: @jakirkham Done.

**Pull request merged**